aiida_pseudo/groups/mixins/cutoffs.py

```diff
@@ -190,12 +185,11 @@ def get_cutoffs(self, stringency=None) -> dict:
         :raises ValueError: if no stringency is specified and no default stringency is defined for the family.
         :raises ValueError: if the requested stringency is not defined for this family.
         """
-        stringency = stringency or self.get_default_stringency()
-
-        try:
-            return self._get_cutoffs_dict()[stringency]
-        except KeyError as exception:
-            raise ValueError(f'stringency `{stringency}` is not defined for this family.') from exception
+        if stringency is None:
+            stringency = self.get_default_stringency()
+        else:
+            self.validate_stringency(stringency)
```

Comment:
I would say that code like this is another argument in favor of `validate_stringency` accepting `None` and just calling `get_default_stringency` in that case.

Reply:
Yes, I agree! Muy limpio!